src/main/java/com/gooddata/account/Account.java

```diff
@@ -126,6 +129,38 @@ public String getId() {
         return getId(getUri());
     }
 
+    public String[] getIpWhitelist() {
+        return ipWhitelist;
+    }
+
+    public void setEmail(final String email) {
+        this.email = email;
+    }
+
+    public void setPassword(final String password) {
+        this.password = password;
+    }
+
+    public void setVerifyPassword(final String verifyPassword) {
+        this.verifyPassword = verifyPassword;
+    }
+
+    public void setFirstName(final String firstName) {
+        this.firstName = firstName;
+    }
+
+    public void setLastName(final String lastName) {
+        this.lastName = lastName;
+    }
+
+    public void setIpWhitelist(final String[] ipWhitelist) {
+        this.ipWhitelist = ipWhitelist;
+    }
+
+    AccountUpdate createAccountUpdate() {
+        return new AccountUpdate(email, password, verifyPassword, firstName, lastName, ipWhitelist);
```

Comment:
some attributes are "read-only" (like login) and cannot be send in body of update request (it will return 4xx). Therefore here I am creating new update entity which does not contain these read-only attributes.  Do you see better solution?